kvrocks.conf

```diff
@@ -96,6 +96,15 @@ dir /tmp/kvrocks
 # We also can send logs to stdout/stderr is as simple as:
 #
 log-dir stdout
+# log-dir log
+
+# You can configure logcleanerday to control whether to enable logcleaner
+# and the maximum number of days that the INFO level logs will be kept.
+# if set logcleanerday to -1 , this mean disable logcleaner.
+# if set logcleanerday between 0 to INT_MAX , this mean enable logcleaner ,
+# and INFO level log file whose last modified time is greater than logcleanerday will be unlinked.
+# By default the logcleanerday is -1.
+logcleanerday -1
```

Comment:
What does 0 mean？

Reply:
how about renaming it to `log-retention-days`

Reply:
if set to 0 , during the flush, the previous INFO level logs will be immediately unlinked

Reply:
So it means won't retent any logs except the current one？

Reply:
yes ，but only INFO level will be unlinked。WARNING，ERROR，FATAL level log will be kept

Reply:
Got it, thanks for your explanation.

Reply:
> yes ，but only INFO level will be unlinked。WARNING，ERROR，FATAL level log will be kept

Sorry ,  this describe is wrong.

I went to see the code of blog . I find that it will unlink all level log.

In the glog , every time a log is written, it will be judged whether to do logclean.

glog set a variable FLAGS_logcleansecs to control the time interval of logclean operation (Default is 5 minutes)。

Each level has a logfileobject , but only one logclean object. So if INFO level logfileobject

do logclean at time 0 , other level logfileobject can only logclean after five minutes.

the code of glog
https://github.com/google/glog/blob/master/src/logging.cc#:~:text=void%20LogCleaner%3A%3AUpdateCleanUpTime,UpdateCleanUpTime()%3B

Reply:
Thanks for your re-claimed, it'd be better if it could remove all level logs instead of only the INFO level.